proxy/envoy/testdata/rds-ingress-ssl.json.golden

```diff
@@ -11,7 +11,7 @@
       "cluster": "out.81c187d71467b1608736c57bb0734f9ef9b68f7d",
       "opaque_config": {
        "mixer_control": "on",
-       "mixer_forward": "off"
+       "mixer_forward": "on"
```

Comment:
It may not work.  In the current code,  we set the special HTTP header first,  then extract its attributes and remove it later.   I need to switch the order around.